Lusas_Adapter/CRUD/Read/Results/BarResults.cs

```diff
@@ -141,236 +212,235 @@ private IEnumerable<IResult> ExtractBarForce(List<int> ids, List<int> loadcaseId
                         mZ * forceSIConversion * lengthSIConversion
                         );
                     barForces.Add(barForce);
-
                 }
-
-                d_LusasData.stopUsingScriptedResults();
-                d_LusasData.flushScriptedResults();
             }
 
+            d_LusasData.stopUsingScriptedResults();
+            d_LusasData.flushScriptedResults();
+
             return barForces;
         }
 
-        /***************************************************/
-
-        private IEnumerable<IResult> ExtractBarStress(List<int> ids, List<int> loadcaseIds)
-        {
-            List<BarStress> barStresses = new List<BarStress>();
+    /***************************************************/
 
-            IFView view = m_LusasApplication.getCurrentView();
-            IFResultsContext resultsContext = m_LusasApplication.newResultsContext(view);
-
-            string entity = "Stress - Thick 3D Beam";
-            string location = "Feature extreme";
-
-            foreach (int loadcaseId in loadcaseIds)
-            {
-                IFLoadset loadset = d_LusasData.getLoadset(loadcaseId);
+    private IEnumerable<IResult> ExtractBarStress(List<int> ids, List<int> loadcaseIds)
```

Comment:
Should anything be changing below this point?

Reply:
Nothing should change below line 218, I don't know how to remove the "changes" but all code should be the same and still works the same as before